### PR TITLE
Lua menu title color fix

### PR DESCRIPTION
**Pull request by bike797**
In EdgeTx 2.5.

Lua folder(include function bind, wifi, joystic ..) menu color is confilict with TITLE_BGCOLOR.

Before
before20210905_012949

After
after20210905_012530

### CODE REVIEW

**Comment by CapnBry** [comment]
Thanks for this.

@StonedDawg can you add the fixes for folders not appearing on X9D and the title color not matching to this as well so we can get them all fixed in one go?

File: src/lua/color_display_480x272/TOOLS/elrsV2.lua [hunk 1]
```diff
@@ -474,7 +474,7 @@ local function lcd_title()
   lcd.drawFilledRectangle(0, 0, LCD_W, 30, TITLE_BGCOLOR)
   local title = (allParamsLoaded == 1 or elrsFlags > 0) and deviceName or "Loading..."
   lcd.drawText(1, 5, title)
```
Comment (maybenikhil) [inline src/lua/color_display_480x272/TOOLS/elrsV2.lua:476]: add a MENU_TITLE COLOR here as well

File: src/lua/color_display_480x272/TOOLS/elrsV2.lua [hunk 2]
```diff
@@ -341,7 +341,7 @@ end
 
 local function fieldFolderDisplay(field,y ,attr)
   lcd.drawFilledRectangle(0, y, LCD_W, 22, TITLE_BGCOLOR)
```
Comment (StonedDawg) [inline src/lua/color_display_480x272/TOOLS/elrsV2.lua:343]: I think we should just remove to keep consistency with the LUA for other devices that has a problem that they are too over-highlighting.

File: src/lua/color_display_480x272/TOOLS/elrsV2.lua [hunk 3]
```diff
@@ -364,8 +364,8 @@ local function fieldCommandSave(field)
 end
 
 local function fieldCommandDisplay(field, y, attr)
-  lcd.drawFilledRectangle(0, y*22+10, LCD_W, 22, TITLE_BGCOLOR)
-  lcd.drawText(10, y, "[" .. field.name .. "]", attr)
+  lcd.drawFilledRectangle(0, y, LCD_W, 22, TITLE_BGCOLOR)
```
Comment (StonedDawg) [inline src/lua/color_display_480x272/TOOLS/elrsV2.lua:367]: this too with my previous comment

**Comment by bike797** [comment]
I'm new to using git
The wrong merge was selected and I don't know how to cancel it.
Sorry for messing up the commit...

somebody fix it please

**Comment by pkendall64** [comment]
Merging master into your branch is fine. It won't mess anything up.

**Comment by StonedDawg** [comment]
@bike797 yep you're fine, can you however resolve the reviews or should I do it later?

**Comment by StonedDawg** [comment]
@bike797 @CapnBry if there is no more fix for the .LUA side, I think this is ready.

**Comment by CapnBry** [comment]
Thanks, both of you!
I've made a small change to the X9D lua so the background of the good/bad isn't inverse and it matches the other title screens everywhere else (like `lcd.drawScreenTitle` does, but that doesn't work if you pass zeroes).

**Review by CapnBry — APPROVED** [review]

**Review by StonedDawg — APPROVED** [review]

**Comment by CapnBry** [comment]
@bike797 There's a problem with the changes you made to the color script. On EdgeTX 2.4 6ded23f2
```
Line 475: attempt to perform arithmetic on field MENU_TITLE_COLOR
```
Can you have a look and figure this out? If not we'll just revert it back to remove the MENU_TITLE_COLOR bit. I had the same issue when I made the changes originally and had to just remove it because I couldn't figure out why it didn't work.

**Comment by bike797** [comment]
Cay you check with this? (about OS Theme Colors, EdgeTX2.5)
#https://github.com/EdgeTX/edgetx/pull/540

**Comment by CapnBry** [comment]
> Cay you check with this? (about OS Theme Colors, EdgeTX2.5)
> #[EdgeTX/edgetx#540](https://github.com/EdgeTX/edgetx/pull/540)

I'm just passing on info from the testing in Discord. I do not have this hardware. Is there a version of EdgeTX that does not have `MENU_TITLE_COLOR`? maybenikhil also reported this error when I did my initial implementation which is why the color was removed from what got merged. If it was missing for some nightlies that's fine but if there was a release version that doesn't have them we need to code around it instead of crashing with like a `(MENU_TITLE_COLOR or XXX)` where XXX is the other working name.

**Comment by bike797** [comment]
I checked some os version with TX16s.
 1. OpenTX 2.3.12-otx. It is ok
 2. EdgeTX2.4 release 09a6580. MENU_TITLE_COLOR error 
 3. EdgeTX2.5? recent master branch (after SEP???). It is ok

we can see EdgeTX LUA Reference Guide about color.
#[EdgeTX Lua Reference Guide](https://luadoc.edgetx.org/part_iii_-_opentx_lua_api_reference/constants/color-constants)

> TITLE_BGCOLOR -> COLOR_THEME_SECONDARY1
MENU_TITLE_COLOR -> COLOR_THEME_PRIMARY2

**Pull request merged**